packages/app-store/googlecalendar/lib/CalendarService.ts

```diff
@@ -121,6 +122,22 @@ export default class GoogleCalendarService implements Calendar {
               console.error("There was an error contacting google calendar service: ", err);
               return reject(err);
             }
+
```

Comment:
After event has been created - we updated it in the creation call back to update the description. We do this here cause we know the created events hangout URL as it is passed in the callback function